exercises/poker/poker_test.exs

```diff
@@ -0,0 +1,215 @@
+if !System.get_env("EXERCISM_TEST_EXAMPLES") do
+  Code.load_file("poker.exs", __DIR__)
+end
+
+ExUnit.start
+ExUnit.configure exclude: :pending, trace: true
+
+defmodule PokerTest do
+  use ExUnit.Case
+
+  #@tag :pending
+  test "single hand always wins" do
+    high_of_jack = ~w(4S 5S 7H 8D JC)
+    assert [high_of_jack] == Poker.best_hand([high_of_jack])
+  end
+
+  @tag :pending
+  test "highest card numeric only" do
+    high_of_8 = ~w(4D 5S 6S 8D 3C)
+    high_of_10 = ~w(2S 4C 7S 9H 10H)
+    assert [high_of_10] == Poker.best_hand([high_of_8, high_of_10])
+  end
+
```

Comment:
I would like to see a test here that passes more than 2 hands. Most of the tests pass only 2 and it is not clear that the user is required to handle more than that until the very last test.

I think it also might be useful to demonstrate earlier that there can be ties.